libs/@guardian/libs/src/consent-management-platform/usnat/api.ts

```diff
@@ -1,19 +1,13 @@
-import type { GPPData } from '../types/usnat';
+import type { UsnatData } from '../types/usnat';
 
-type Command = 'ping';
-
-const api = (command: Command) =>
-	new Promise((resolve, reject) => {
-		if (window.__gpp) {
-			window.__gpp(command, (result, success) =>
-				success
-					? resolve(result)
-					: /* istanbul ignore next */
-						reject(new Error(`Unable to get ${command} data`)),
-			);
-		} else {
```

Comment:
Should we keep this else? Seems like this promise might never resolve if `getUserConsents` doesn't exist on the window?